jbmc/src/java_bytecode/CMakeLists.txt

```diff
@@ -12,4 +12,4 @@ generic_includes(java_bytecode)
 
 # if you link java_bytecode.a in, then you also need to link other .a libraries
 # in
-target_link_libraries(java_bytecode util goto-programs miniz json)
+target_link_libraries(java_bytecode util goto-programs miniz json ansi-c)
```

Comment:
This cannot be right.

Reply:
I'm afraid it can: expr2java depend builds upon expr2c. It was just previously hidden by the fact that `goto-programs` already provided the link to `ansi-c`.

Reply:
Splitting these up is on the todo-list (meaning there should be some factoring out of what's common, and it should be done using the new format_expr stuff).